tests/logger/LogHelper.test.ts

```diff
@@ -32,22 +33,18 @@ const graphQLErrorWithVariables: GraphQLError = new GraphQLError(messageWithVari
     undefined,
     undefined
     , {exception: 'A stacktrace', query: customerQuery, serviceName: 'myTestService'})
-const graphQLErrorWithSourceBody: GraphQLError =
-    new GraphQLError('`CustomerPayload` is an extension type',
-        undefined,
-        {locationOffset: {line: 1, column: 1}, name: 'doesnotmatter', body: customerQuery},
-        undefined,
-        undefined,
-        undefined
-        , {exception: 'A stacktrace', serviceName: 'customer'})
-const graphQLErrorWithAstNode: GraphQLError =
-    new GraphQLError('`CustomerPayload` is an extension type',
-        {kind: 'NamedType', loc: undefined, name: { kind: 'Name', value: 'customer'}},
-        undefined,
-        undefined,
-        undefined,
-        undefined
-        , {exception: 'A stacktrace', serviceName: 'customer'})
+const graphQLErrorWithSourceBody = new GraphQLError('`CustomerPayload` is an extension type',{
+    source: {get [Symbol.toStringTag](): string {
+        return this[Symbol.toStringTag]
+    }, locationOffset: {line: 1, column: 1}, name: 'doesnotmatter', body: customerQuery},
+    extensions: {exception: 'A stacktrace', serviceName: 'customer'}
```

Comment:
Can you add some line breaks to this constructor? It's a bit hard to grasp which brackets belong together.

Reply:
Done